generated/storage/labels.pb.go

```diff
@@ -92,7 +92,7 @@ func (SetBasedLabelSelector_Operator) EnumDescriptor() ([]byte, []int) {
 
 // Label selector components are joined with logical AND, see
 //
-//	https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+//     https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
```

Comment:
So for reference when I run `make proto-generated-sources`, this is the list of changed files that I'm jut not including because of the style checks failing:
```
	modified:   generated/api/v1/authprovider_service.pb.go
	modified:   generated/api/v1/debug_service.pb.go
	modified:   generated/internalapi/central/sensor_events.pb.go
	modified:   generated/storage/cluster.pb.go
	modified:   generated/storage/group.pb.go
	modified:   generated/storage/labels.pb.go
	modified:   generated/storage/network_policy.pb.go
	modified:   generated/storage/node.pb.go
	modified:   generated/storage/notifier.pb.go
	modified:   generated/storage/rbac.pb.go
	modified:   generated/storage/secret.pb.go
	modified:   generated/storage/service_account.pb.go
	modified:   generated/storage/test.pb.go
	modified:   generated/test/test.pb.go

```